src/js/components/HeaderBar.tsx

```diff
@@ -1,20 +1,67 @@
 import * as React from "react";
+import AuthStore from "#SRC/js/stores/AuthStore";
+
 import { HeaderBar as UIHeaderBar } from "ui-kit-stage/HeaderBar";
+import AccountDropdown from "./AccountDropdown";
+
+function getUserAccountDropdown() {
+  const menuItems = [
+    {
+      html: "Logout",
+      id: "logout",
+      onClick: AuthStore.logout
+    }
+  ];
+
+  var userLabel = AuthStore.getUserLabel();
```

Comment:
Can we use `const` instead of `var` since we re not mutating userLabel.

Reply:
yep, good catch